pkgs/applications/window-managers/sway/lock-effects.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkgconfig, scdoc
+, wayland, wayland-protocols, libxkbcommon, cairo, gdk-pixbuf, pam
+}:
+
+stdenv.mkDerivation rec {
+  pname = "swaylock-effects";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "mortie";
+    repo = "swaylock-effects";
+    rev = "v${version}";
```

```suggestion
    rev = "v1.6-0";
```

Tag is actually `v1.6-0` for whatever reason.

---

Then maybe `version` should be changed instead?

---

Changed `version` to `v1.6-0`